ios/default.nix

```diff
@@ -278,7 +278,74 @@ nixpkgs.runCommand "${executableName}-app" (rec {
     # focus????
     focus/reflex-platform/scripts/run-in-ios-sim "$tmpdir/${executableName}.app"
   '';
-}) ''
+  portableDeployScript = nixpkgs.writeText "portable-deploy" ''
+    #!/usr/bin/env bash
+    set -eo pipefail
+
+    if (( "$#" < 1 )); then
+      echo "Usage: $0 [TEAM_ID]" >&2
+      exit 1
+    fi
+
+    TEAM_ID=$1
+    shift
+
+    set -euo pipefail
+
+    function cleanup {
+      if [ -n "$tmpdir" -a -d "$tmpdir" ]; then
+        echo "Cleaning up tmpdir" >&2
+        chmod -R +w $tmpdir
+        rm -fR $tmpdir
+      fi
+    }
+
+    trap cleanup EXIT
+
+    tmpdir=$(mktemp -d)
+    # Find the signer given the OU
+    signer=$({ security find-certificate -c 'iPhone Developer' -a; security find-certificate -c 'Apple Development' -a; } \
+      | grep '^    "alis"<blob>="' \
+      | sed 's|    "alis"<blob>="\(.*\)"$|\1|' \
+      | while read c; do \
+          security find-certificate -c "$c" -p \
+            | ${nixpkgs.libressl}/bin/openssl x509 -subject -noout; \
+        done \
+      | grep "OU[[:space:]]\?=[[:space:]]\?$TEAM_ID" \
+      | sed 's|subject= /UID=[^/]*/CN=\([^/]*\).*|\1|' \
+      | head -n 1 || true)
+
+    if [ -z "$signer" ]; then
+      echo "Error: No iPhone Developer certificate found for team id $TEAM_ID" >&2
+      exit 1
+    fi
+
+    dir="$tmpdir/${executableName}-${bundleVersion}"
+    mkdir $dir
+
+    cp -LR "$(dirname $0)/../${executableName}.app" $dir
```

Comment:
This seems a bit "presumptuous". Maybe we could take it as a param with a default?